cmd/push.go

```diff
@@ -17,11 +17,16 @@ var pushCommand = &cobra.Command{
 	Long: `Push a custom rules bundle to an OCI registry.
 
 To start, log into your OCI registry locally and then run:
-$ snyk-iac-rules push <path to generated bundle> --registry <container registry>
+$ snyk-iac-rules push <path to generated bundle> --registry <desired location of OCI artifact in your OCI registry>
 If the tag is not provided at the end of the provided registry, the tool defaults to 'latest'.
 
-The 'push' command takes in the container registry and pushes the bundle at the provided 
-path. The container registry must support OCI artifacts.
+For example, if your generated bundle is called bundle.tar.gz and the OCI registry you're using is DockerHub, the following command must be run: 
```

Comment:
extremely minor nit:
maybe we should format the bundle.tar.gz as \`bundle.tar.gz\`?